src/parser/jobs/blm/RotationWatchdog.js

```diff
@@ -53,21 +60,44 @@ export default class RotationWatchdog extends Module {
 	//snapshot buffs and UH at the beginning of your recording
 	_onBegin(event) {
 		const actionId = event.ability.guid
+
+		//get UI status for to check for T3
+		this._UI = this.gauge.getUI()
+		this._AF = this.gauge.getAF()
 		if (actionId === ACTIONS.FIRE_III.id) {
 			this._lockingBuffs()
-		}
+		//Check to see if we get a T3 > F3
```

Comment:
This belongs really to the next if statement rather than in the middle of nowhere here.
The next line isn't even the check, it's just the state reset so the check can operate.